eth-bytecode-db/eth-bytecode-db/src/verification/db.rs

```diff
@@ -95,26 +136,14 @@ async fn insert_files(
 ) -> Result<Vec<files::Model>, anyhow::Error> {
     let mut result = Vec::new();
     for (name, content) in files {
-        let file = {
-            let file = files::Entity::find()
-                .filter(files::Column::Name.eq(name.clone()))
-                .filter(files::Column::Content.eq(content.clone())) // TODO: I believe it is quite expensive to search by the content
-                .one(txn)
-                .await
-                .context("select from \"files\" by \"name\" and \"content\"")?;
-
-            match file {
-                Some(file) => file,
-                None => files::ActiveModel {
-                    name: Set(name),
-                    content: Set(content),
-                    ..Default::default()
-                }
-                .insert(txn)
-                .await
-                .context("insert into \"files\"")?,
-            }
+        let active_model = files::ActiveModel {
+            name: Set(name.clone()),
+            content: Set(content.clone()),
+            ..Default::default()
         };
+        let (file, _inserted) =
```

Comment:
Nit: let (file, _) ?

Reply:
Prefer the version with direct indication what the second argument returns